test/unit/uk/gov/hmrc/exports/services/mapping/governmentagencygoodsitem/GovernmentAgencyGoodsItemBuilderSpec.scala

```diff
@@ -77,15 +77,25 @@ class GovernmentAgencyGoodsItemBuilderSpec
 
           verify(statisticalValueAmountBuilder)
             .buildThenAdd(refEq(exportItem), any[GoodsShipment.GovernmentAgencyGoodsItem])
-          verify(packagingBuilder).buildThenAdd(refEq(exportItem), any[GoodsShipment.GovernmentAgencyGoodsItem])
-          verify(governmentProcedureBuilder).buildThenAdd(refEq(exportItem), any[GoodsShipment.GovernmentAgencyGoodsItem])
-          verify(additionalInformationBuilder).buildThenAdd(refEq(exportItem), any[GoodsShipment.GovernmentAgencyGoodsItem])
-          verify(additionalDocumentsBuilder).buildThenAdd(refEq(exportItem), any[GoodsShipment.GovernmentAgencyGoodsItem])
-          verify(commodityBuilder).buildThenAdd(any[Commodity], any[GoodsShipment.GovernmentAgencyGoodsItem])
-          verify(mockCachingMappingHelper).mapGoodsMeasure(any[CommodityMeasure])
-          verify(mockCachingMappingHelper).commodityFromExportItem(any[ExportItem])
+          verify(packagingBuilder)
```

Comment:
tbh...
```
verify(governmentProcedureBuilder).buildThenAdd(refEq(exportItem), any[GoodsShipment.GovernmentAgencyGoodsItem])
```
is more readable than
```
verify(governmentProcedureBuilder)
  .buildThenAdd(refEq(exportItem), any[GoodsShipment.GovernmentAgencyGoodsItem])
```

Reply:
I'd changed it as there were some existing lines too long so were split and others on one line so it was easier to read when having all uniform, but it makes little difference so I can change it back